nasa/index.md

```diff
@@ -13,15 +13,15 @@ photos:
 
```

Comment:
You should probably remove this file

Reply:
Thanks! I deleted the `nasa/index.md` but didn't commit it. So, it didn't reflect.

I think that's fixed now...